generators/client-2/templates/src/main/webapp/app/home/home.html

```diff
@@ -4,7 +4,7 @@
             <span class="hipster img-responsive img-rounded"></span>
         </div>
         <div class="col-md-8">
-            <h1 translate="home.title">Welcome, Java Hipster!</h1>
+            <h1 [jhi-translate]="'home.title' | translate">Welcome, Java Hipster!</h1>
```

Comment:
`pipe` is following ng2-translate.
`directive` manually done.

They have a method inside But I don't want to use that and re-engineer the thing.

The api needs `single-quote`.